hono/src/utils/swaggerSchema.ts

```diff
@@ -0,0 +1,300 @@
+import * as fs from 'fs'
+import {
+  OpenAPIRegistry,
+  OpenApiGeneratorV3,
+  extendZodWithOpenApi,
+} from '@asteasolutions/zod-to-openapi'
+import { z } from 'zod'
+
+extendZodWithOpenApi(z)
+
+const registry = new OpenAPIRegistry()
+
+const ItemIdSchema = registry.registerParameter(
+  'ItemId',
+  z.string().openapi({
+    param: {
+      name: 'id',
+      in: 'path',
+    },
+    example: '1212121',
+  })
+)
+
+const ItemIdSchemaDel = registry.registerParameter(
+  'ItemId',
+  z.string().openapi({
+    param: {
+      name: 'id',
+      in: 'path',
+    },
+    example: '1212121',
+  })
+)
+
+const OutfitIdSchema = registry.registerParameter(
+  'ItemId',
+  z.string().openapi({
+    param: {
+      name: 'id',
+      in: 'path',
+    },
+    example: '1212121',
+  })
+)
+
+const OutfitIdSchemaDel = registry.registerParameter(
+  'ItemId',
+  z.string().openapi({
+    param: {
+      name: 'id',
+      in: 'path',
+    },
+    example: '1212121',
+  })
+)
+
+// Correcting schema definitions and registrations:
+const ItemSchema = z.object({
```

Comment:
Make sure you have the right schema definitions. For example, in `schema,ts`, we have more attributes present for `items`: name, brand, photoUrl, type, etc. The same goes for `outfits`.